src/Doc/Masa.Blazor.Doc/Masa.Blazor.Doc.csproj

```diff
@@ -22,13 +22,13 @@
 
 	<ItemGroup>
 	  <Content Update="wwwroot\locale\en-US.json">
-	    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+	    <CopyToOutputDirectory>Always</CopyToOutputDirectory>
 	  </Content>
 	  <Content Update="wwwroot\locale\languageConfig.json">
 	    <CopyToOutputDirectory>Always</CopyToOutputDirectory>
 	  </Content>
 	  <Content Update="wwwroot\locale\zh-CN.json">
-	    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+	    <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Why always?

Reply:
When debugging the code locally, the json file is often an old file, and the compiler does not copy the file when it is updated in time. It is the safest to choose always copy.